torch/csrc/utils/tensor_new.h

```diff
@@ -1,14 +1,20 @@
 #pragma once
 
 #include "torch/csrc/python_headers.h"
-#include "torch/csrc/utils/device.h"
-#include <ATen/ATen.h>
```

Comment:
Is there a reason to remove the header? You can't do anything with `tensor_new.h` without the full Tensor struct definition (i.e. <ATen/ATen.h>)